packages/block-library/src/cover/edit/index.js

```diff
@@ -59,6 +61,82 @@ function getInnerBlocksTemplate( attributes ) {
 	];
 }
 
+/**
+ * Computes the average color of an image.
+ * @param {string} url
+ * @return {Promise<string>} average color in hex
+ */
+async function computeAverageColor( url ) {
+	function retrieveFastAverageColor() {
+		if ( ! retrieveFastAverageColor.fastAverageColor ) {
+			retrieveFastAverageColor.fastAverageColor = new FastAverageColor();
+		}
+		return retrieveFastAverageColor.fastAverageColor;
+	}
```

Comment:
We aren't getting the benefits of caching the FAC instance if we're rebuilding the function every time we call `comuteAverageColor`. This should be a top-level function like before.